GCC7 fixes

error: function 'GfxFontLoc& GfxFontLoc::operator=(GfxFontLoc&&)' defaulted on
its redeclaration with an exception-specification that differs from the
implicit exception-specification ''

error: function 'X509CertificateInfo::EntityInfo&
X509CertificateInfo::EntityInfo::operator=(X509CertificateInfo::EntityInfo&&)'
defaulted on its redeclaration with an exception-specification that differs
from the implicit exception-specification ''

--- poppler/GfxFont.cc.orig
+++ poppler/GfxFont.cc
@@ -180,7 +180,7 @@
 
 GfxFontLoc::GfxFontLoc(GfxFontLoc &&other) noexcept = default;
 
-GfxFontLoc &GfxFontLoc::operator=(GfxFontLoc &&other) noexcept = default;
+GfxFontLoc &GfxFontLoc::operator=(GfxFontLoc &&other) = default;
 
 void GfxFontLoc::setPath(GooString *pathA)
 {
--- poppler/GfxFont.h.orig
+++ poppler/GfxFont.h
@@ -124,7 +122,7 @@
     GfxFontLoc(const GfxFontLoc &) = delete;
     GfxFontLoc(GfxFontLoc &&) noexcept;
     GfxFontLoc &operator=(const GfxFontLoc &) = delete;
-    GfxFontLoc &operator=(GfxFontLoc &&other) noexcept;
+    GfxFontLoc &operator=(GfxFontLoc &&other);
 
     // Set the 'path' string from a GooString on the heap.
     // Ownership of the object is taken.
--- poppler/CertificateInfo.cc.orig
+++ poppler/CertificateInfo.cc
@@ -39,7 +39,7 @@
 
 X509CertificateInfo::EntityInfo::EntityInfo(X509CertificateInfo::EntityInfo &&other) noexcept = default;
 
-X509CertificateInfo::EntityInfo &X509CertificateInfo::EntityInfo::operator=(X509CertificateInfo::EntityInfo &&other) noexcept = default;
+X509CertificateInfo::EntityInfo &X509CertificateInfo::EntityInfo::operator=(X509CertificateInfo::EntityInfo &&other) = default;
 
 X509CertificateInfo::X509CertificateInfo() : ku_extensions(KU_NONE), cert_version(-1), is_self_signed(false) { }
 
--- poppler/CertificateInfo.h.orig
+++ poppler/CertificateInfo.h
@@ -70,7 +70,7 @@
         ~EntityInfo();
 
         EntityInfo(EntityInfo &&) noexcept;
-        EntityInfo &operator=(EntityInfo &&) noexcept;
+        EntityInfo &operator=(EntityInfo &&);
 
         EntityInfo(const EntityInfo &) = delete;
         EntityInfo &operator=(const EntityInfo &) = delete;
